scss/_offcanvas.scss

```
@@ -0,0 +1,40 @@
+.offcanvas {
+  position: fixed;
+  top: 0;
+  bottom: 0;
+  left: 0;
+  z-index: $zindex-offcanvas;
+  width: $offcanvas-horizontal-width-mobile;
+  max-width: 100%;
+  overflow-y: auto;
+  @include transition(transform $offcanvas-transition-duration ease-in-out);
+  transform: translateX(-100%);
+}
+
+.offcanvas-right {
+  right: 0;
+  left: auto;
+  transform: translateX(100%);
+}
+
+.offcanvas-bottom {
+  top: auto;
+  right: 0;
+  width: 100%;
+  height: $offcanvas-vertical-height-mobile;
+  transform: translateY(100%);
+}
+
+@include media-breakpoint-up($offcanvas-mobile-breakpoint) {
+  .offcanvas:not(.offcanvas-bottom) {
+    width: $offcanvas-horizontal-width;
+  }
+  .offcanvas-bottom {
+    height: $offcanvas-vertical-height;
+  }
+}
+
+.offcanvas.show {
+  transform: translate(0, 0);
```

This can be simplified:
```suggestion
  transform: none;
```